### PR TITLE
Bookstore Styling

### PR DESCRIPTION
**Pull request by tajemouti**
## Hi Reviewer 👋 ##
Below is a list of the accomplished tasks while working on this milestone:

1. Created a feature branch for this milestone.
2. Styled the navigation bar.
3. Added buttons to the book list elements.
4. Styled the book list elements.
5. Styled the form element.
6. Fixed linters' errors.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
+  <>
+    <header>
+      <nav className="navbar">
+        <h2 className="logo">Bookstore CMS</h2>
+        <ul>
+          <li className="books-link">
+            <NavLink to="/">BOOKS</NavLink>
+          </li>
+          <li className="categories">
+            <NavLink to="/categories">CATEGORIES</NavLink>
```
Comment (t-yanick) [inline src/components/Header.js:14]: - [ ] As per Zeplin design, you have not respected the text color of your Navbar tabs. Kindly make the color of Books and Categories same as in the Zeplin design

File: src/components/Book.js [hunk 2]
```diff
@@ -13,28 +13,38 @@ const Book = (props) => {
...
-          <span>Author :</span>
-          {author}
-        </p>
+      <div className="reading-progress">
+        <div className="progress" />
+        <div className="progress-percentage">
+          <span className="percentage">76%</span>
+          <span className="completed">Completed</span>
+        </div>
       </div>
```
Comment (t-yanick) [inline src/components/Book.js]: - [ ] As per Zeplin design, there is a vertical rule between the percentage completion and the Chapter number as seen in the screenshot below. This vertical rule is missing in your design
Screen Shot 2023-09-02 at 6 12 19 AM

File: src/components/Book.js [hunk 3]
```diff
@@ -13,28 +13,38 @@ const Book = (props) => {
...
-      <div>
-        <p>
-          <span>Category :</span>
-          {category}
-        </p>
+      <div className="current-chapter">
+        <span className="heading">CURRENT CHAPTER</span>
+        <span className="chapter">Chapter 17</span>
+        <button type="button" className="update-btn">UPDATE PROGRESS</button>
```
Comment (t-yanick) [inline src/components/Book.js]: - [ ] Please kindly check all your **font types and colors** to ensure that they all look **EXACTLY** like the Zeplin design. I see some differences which I cannot individually highlight here. But kindly check all the text on your page to ensure that they conform to **font** **type** and **color**.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @tajemouti , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Use of correct Gitflow
:heavy_check_mark: No linter errors

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 3 above)
Comment (tajemouti) [reply src/components/Book.js]: @t-yanick,
Thank you for the insightful feedback. I really appreciate it.
Cheers

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @tajemouti, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  Implemented required changes. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**